include/mxnet/c_predict_api.h

```diff
@@ -43,8 +43,6 @@ extern "C" {
 
 /*! \brief manually define unsigned int */
 typedef uint32_t mx_uint;
```

Comment:
Why do we still need this?

Reply:
yes we do other language bindings use this

Reply:
Which language bindings use this?

Reply:
java, C++, R, Scala

Reply:
https://github.com/apache/incubator-mxnet/blob/3f3ba92ae1468d08de088d2291ca14e2d5dc5515/perl-package/AI-MXNetCAPI/mxnet_typemaps.i#L171

https://github.com/apache/incubator-mxnet/blob/3f3ba92ae1468d08de088d2291ca14e2d5dc5515/R-package/src/ndarray.cc#L34

https://github.com/apache/incubator-mxnet/blob/b88705e7744be1cc22122eb2c5b492e495ab45bb/scala-package/native/src/main/native/org_apache_mxnet_native_c_api.cc#L89